### ISSUE TITLE
[AEM 6.4 + SP2] - Jackson Dependency Issue

### ISSUE THREAD

**Issue by godanny86** [adobe/aem-spa-project-archetype#23]
AEM version: AEM 6.4.2

The **Core** bundle relies on a dependency on `com.fasterxm.jackson.*` version **2.9.7**. 

However, on AEM 6.4 + SP2 the dependency included is **2.8.4**. This causes the bundle to fail to start:

```
com.fasterxml.jackson.annotation,version=[2.9,3) -- Cannot be resolved
```
I understand there is a security issue with version 2.8.4, perhaps the latest version of the bundle can be embedded in the project? Or maybe in AEM 6.4 + SP3 this dependency can be updated?

**Comment by lydiapuric** [comment]
Thanks Gordon for your feedback.
Unfortunately com.fasterxm.jackson 2.8.4 has a security vulnerability, see http://www.cvedetails.com/vulnerability-list.php?vendor_id=15866&product_id=42991&version_id=237509&page=1&hasexp=0&opdos=0&opec=0&opov=0&opcsrf=0&opgpriv=0&opsqli=0&opxss=0&opdirt=0&opmemc=0&ophttprs=0&opbyp=0&opfileinc=0&opginf=0&cvssscoremin=0&cvssscoremax=0&year=0&month=0&cweid=0&order=1&trc=3&sha=1e493f7b77ddee795faddfb91c709c2c5272cb4c. This should be addressed in AEM 6.4 latest SP, I will check for the status and investigate either in documentation or embedding the dependency.

**Comment by lydiapuric** [comment]
Decision has been taken: com.fasterxml.jackson dependencies with match the 6.4 dependencies from SP3 and therefore set back to 2.8.4.
We will push for an update.
